tests/testsuite/update.rs

```diff
@@ -1950,6 +1989,7 @@ fn update_breaking() {
                 yanked  =  "1.0"  # Comment
                 ws.workspace  =  true  # Comment
                 shared  =  "2.0"  # Comment
+                multiple-locations  =  { path  =  "../multiple-locations", version  =  "1.0" }  # Comment
```

Comment:
@epage I'm wondering how you feel about this version not getting upgraded.

Reply:
Leaving this alone is the correct behavior.  Cargo will error out if the `version` req does not match the version at `path`.